common/djangoapps/course_category/tests/test_search_engine.py

```diff
@@ -0,0 +1,43 @@
+
+import pytest
+from course_category.models import CourseCategory, Program
```

Comment:
Imports ordering please - blank line between third party and internal.

https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#imports-order